### PR TITLE
Feature project viz

### PR DESCRIPTION
**Pull request by bibekdahal**

### CODE REVIEW

File: config/webpack.dev.js [hunk 1]
```diff
@@ -35,6 +35,7 @@ module.exports = (env) => {
                 'base-scss': path.resolve(appBase, 'src/stylesheets/'),
                 'rs-scss': path.resolve(appBase, 'src/vendor/react-store/stylesheets/'),
             },
+            symlinks: false,
```
Comment (tnagorra) [inline config/webpack.dev.js:38]: is this value not required in webpack.prod.js?

File: src/views/Project/Details/General/Dashboard/index.js [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+        response: PropTypes.object,
+    }).isRequired,
+};
+
+const defaultProps = {
+    className: '',
+};
+
+
+// eslint-disable-next-line no-underscore-dangle
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:29]: This is very clean.
We should move this to utils and have a better doc for it

File: src/views/Project/Details/General/Dashboard/index.js [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+    )
+
+    handleRegionChange = (selectedRegion) => {
+        this.setState({ selectedRegion });
+    }
+
+    renderLeadsActivity = () => (
+        <div className={styles.chart}>
+            <h4>
+                Leads Activity - Last 7 Days
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:80]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+                yLabelModifier={ProjectDashboard.leadsActivityNumberModifier}
+                fill
+            />
+        </div>
+    )
+
+    renderEntriesActivity = () => (
+        <div className={styles.chart}>
+            <h4>
+                Entries Activity - Last 7 Days
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:97]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+            />
+        </div>
+    )
+
+    renderSourcers = () => {
+        const { projectRequest: { response: { topSourcers, numberOfLeads } } } = this.props;
+
+        return (
+            <div className={styles.userTable}>
+                <h4> Top sourcers </h4>
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:116]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 6]
```diff
@@ -0,0 +1,301 @@
...
+        return (
+            <div className={styles.summary}>
+                <div className={styles.summaryItem}>
+                    <Numeral
+                        className={styles.value}
+                        precision={0}
+                        value={project.numberOfLeads}
+                    />
+                    <div className={styles.label}>
+                        Total leads
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:167]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 7]
```diff
@@ -0,0 +1,301 @@
...
+                    </div>
+                </div>
+                <div className={styles.summaryItem}>
+                    <Numeral
+                        className={styles.value}
+                        precision={0}
+                        value={project.numberOfEntries}
+                    />
+                    <div className={styles.label}>
+                        Total entries
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:177]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 8]
```diff
@@ -0,0 +1,301 @@
...
+                    </div>
+                </div>
+                <div className={styles.summaryItem}>
+                    <Numeral
+                        className={styles.value}
+                        precision={0}
+                        value={project.numberOfUsers}
+                    />
+                    <div className={styles.label}>
+                        Total users
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:187]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 9]
```diff
@@ -0,0 +1,301 @@
...
+    }
+
+    renderInfo = () => {
+        const { projectRequest: { response: project } } = this.props;
+
+        return (
+            <div className={styles.info}>
+                <div className={styles.infoItem}>
+                    <div className={styles.label}>
+                        Project status
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:201]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 10]
```diff
@@ -0,0 +1,301 @@
...
+                    </div>
+                    <div className={styles.value}>
+                        {project.status}
+                    </div>
+                </div>
+                <div className={styles.infoItem}>
+                    <div className={styles.label}>
+                        Created by
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:209]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 11]
```diff
@@ -0,0 +1,301 @@
...
+                    </div>
+                    <div className={styles.value}>
+                        {project.createdBy}
+                    </div>
+                </div>
+                <div className={styles.infoItem}>
+                    <div className={styles.label}>
+                        Created at
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:217]: translation should not be skipped

File: src/views/Project/Details/General/Dashboard/index.js [hunk 12]
```diff
@@ -0,0 +1,301 @@
...
+        </div>
+    )
+
+    renderMap = () => {
+        const { projectRequest: { response: { regions } } } = this.props;
+        if (regions.length === 0) {
+            return (
+                <div className={_cs(styles.row, styles.map)}>
+                    <Message>
+                        There is no geo region for this project.
```
Comment (tnagorra) [inline src/views/Project/Details/General/Dashboard/index.js:258]: translation should not be skipped

File: config/webpack.dev.js (hunk 1 above)
Comment (frozenhelium) [reply config/webpack.dev.js:38]: No

**Pull request closed without merging**